### PR TITLE
Django 2.0 support

### PR DESCRIPTION
**Pull request by jpic**
firstttttttttttttttttttttt

With <3 <3 <3

### CODE REVIEW

File: autoslug/settings.py [hunk 1]
```diff
@@ -58,7 +58,11 @@
 
 """
 from django.conf import settings
-from django.core.urlresolvers import get_callable
+
+try:
+    from django.urls import get_callable
+except ImportError:
```
Comment (codingjoe) [inline autoslug/settings.py]: Maybe it makes sense to leave a comment or todo to remove this exception handling when the version is removed.
This is for really old versions only anyhow. Django will drop support for anything prio to 1.11 once 2.0 is released.

**Comment by codingjoe** [comment]
@jpic maybe it would make sense to also bump the testing requirements.

File: autoslug/settings.py (hunk 1 above)
Comment (jpic) [reply autoslug/settings.py]: I've read that too but 1.8 LTS seems to have extended support until at least April 2018
https://www.djangoproject.com/download/

**Comment by JayTurnr** [comment]
@neithere Get off your ass and merge this, please.

**Comment by justinmayer** [comment]
Thank you for the contribution. Django 2.x support has been merged via: https://github.com/justinmayer/django-autoslug/commit/4dc75083d84265e019a900d636273c731457193d

**Pull request closed without merging**